README.md

```diff
@@ -62,10 +62,18 @@ Create a virtual environment, activate it and run `make install`.
 
 We use [Jupytext](https://github.com/mwouts/jupytext) to help version
 Jupyter notebooks.
-See in particular the documentation on [paired notebooks](https://jupytext.readthedocs.io/en/latest/#paired-notebooks).
+Each notebook corresponds to a Python script, which is easy to review.
+See also in the Jupytext documentation on
+[paired notebooks](https://jupytext.readthedocs.io/en/latest/#paired-notebooks).
+
```

Comment:
How about we add a contribution section saying what is the process of submitting changes to code & notebooks?

For example when updating notebook, you need to rerun the whole notebook and then run `make pair_notebooks` ?

Reply:
Yep, good suggestion, let me do that!